assets/js/common/TimeSeriesLineChart/TimeSeriesLineChart.test.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+
+import { render } from '@testing-library/react';
+import TimeSeriesLineChart from './TimeSeriesLineChart';
+import '@testing-library/jest-dom';
+
+describe('TimeSeriesLineChart component', () => {
+  it('should raise an error if the datasets are more then 5', () => {
+    const datasets = [
+      {
+        name: 'Data 1',
+        timeFrames: [{ time: new Date(), value: 0.0 }],
+      },
+      {
+        name: 'Data 2',
+        timeFrames: [{ time: new Date(), value: 0.0 }],
+      },
+      {
+        name: 'Data 3',
+        timeFrames: [{ time: new Date(), value: 0.0 }],
+      },
+      {
+        name: 'Data 4',
+        timeFrames: [{ time: new Date(), value: 0.0 }],
+      },
+      {
+        name: 'Data 5',
+        timeFrames: [{ time: new Date(), value: 0.0 }],
+      },
+      {
+        name: 'Data 6',
+        timeFrames: [{ time: new Date(), value: 0.0 }],
+      },
+      {
+        name: 'Data 7',
+        timeFrames: [{ time: new Date(), value: 0.0 }],
+      },
+    ];
+
+    expect(() =>
+      render(
+        <div>
+          <TimeSeriesLineChart
+            title="test chart"
+            start={new Date()}
```

Comment:
Hey, I'm a bit concerned about `start={new Date()}` and `end={new Date()}`
It could be executed slightly at a slightly different time. 
We could declare a time object outside and reuse it two times, so they are for sure the same :)
What do you think ?